pkg/apis/management.cattle.io/v3/machine_types.go

```diff
@@ -238,6 +238,7 @@ type NodeSpec struct {
 	DesiredNodeUnschedulable string          `json:"desiredNodeUnschedulable,omitempty"`
 	NodeDrainInput           *NodeDrainInput `json:"nodeDrainInput,omitempty"`
 	MetadataUpdate           MetadataUpdate  `json:"metadataUpdate,omitempty"`
+	Scaledown                string          `json:"scaledown,omitempty"`
```

Comment:
maybe it's just me but could we name this better? it sounds like a bool to me 

also, do we want to drop this from api view (similar to desiredNodeUnschedulable) so it's not available for people to edit?

Reply:
Definitely want it in the api, re naming I'm open to suggestions

Reply:
what is the name of this supposed to be?

Reply:
Err. what is the value of this supposed to represent? I agree that the name doesn't immediately indicate what it is.

Reply:
it's an RFC3339 timestamp for when the node will be pulled from the pool. the action sets it to now but you can set it via the api to a time in the future and the nodepool will pull the node after that time

Reply:
So it's the timestamp of when to scale down? 

So could it be `ScaledownTimestamp`? I'm not great at naming items.

Reply:
we do `LastUpdateTime` on norman conditions which are also rfc3339 timestamps, how about `ScaledownTime` ?

Reply:
sounds better than `scaledown`